### PR TITLE
fix error on production mode

### PR DESCRIPTION
**Pull request by bruxx-6243**

### CODE REVIEW

**Comment by bruxx-6243** [comment]
Images rendering on the production mode is set up.

**Comment by bruxx-6243** [comment]
review-requested:[Fariol Blondeau]

File: src/components/ProductCard.astro [hunk 1]
```diff
@@ -14,7 +14,7 @@ const { image, title } = props;
 
 <article {...rest}>
   <Image
-    src={image}
+    src={`/images/${image}`}
```
Comment (bruxx-6243) [inline src/components/ProductCard.astro:17]:
```suggestion
    src={`/images/${image}`} as we know that Astro generate a static site so it was a good idea to pass the images pathname directly here in the compnent than in the data
```

File: src/data/data.ts [hunk 2]
```diff
@@ -5,26 +5,26 @@ export interface IProduct {
...
     title: "Trending on instagram",
-    image: "images/4.png",
+    image: "4.png",
   },
   {
     title: "All Under $40",
-    image: "images/5.png",
+    image: "5.png",
   },
 ];
```
Comment (bruxx-6243) [inline src/data/data.ts:30]: Always use the images name rather using the pathname

**Review by bruxx-6243 — COMMENTED** [review]
All PR are verified

File: src/components/ProductCard.astro (hunk 1 above)
Comment (bruxx6243) [inline src/components/ProductCard.astro:17]: Good

File: src/data/data.ts [hunk 3]
```diff
@@ -5,26 +5,26 @@ export interface IProduct {
...
+    title: "Hoodies & Sweatshirt",
+    image: "1.png",
   },
   {
     title: "Coats & Parkas",
-    image: "images/2.png",
+    image: "2.png",
   },
   {
     title: "Tees & T-Shirt",
```
Comment (bruxx6243) [inline src/data/data.ts:16]: Good job

**Review by bruxx6243 — CHANGES_REQUESTED** [review]

**Pull request merged**